packages/web-twig/src/Resources/components/Header/HeaderMobileActions.twig

```diff
@@ -0,0 +1,26 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _class = props.class | default(null) -%}
+{%- set _dialogId = props.dialogId -%}
+{%- set _menuToggleLabel = props.menuToggleLabel | default('Menu') -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'HeaderMobileActions' -%}
+
+{# Miscellaneous #}
+{%- set _classNames = [ _rootClassName, _class ] -%}
+
+<div {{ mainProps(props) }} {{ classProp(_classNames) }}>
+    {% block content %}{% endblock %}
+    <Button
+        aria-controls="{{ _dialogId }}"
+        aria-expanded="false"
+        color="inverted"
+        data-target="{{ '#' ~ _dialogId }}"
```

Comment:
https://github.com/lmc-eu/spirit-design-system/pull/712/commits/ad5f2504ff038466644847552a9c58e7ad0edf88#diff-1a9d9ea7bedada3ec315723bdeb3c8b3c0ff695ef2ddd8f211846edd482f2f66R18

```suggestion
        data-target="#{{ _dialogId }}"
```

I'm not sure if there's a significant difference between these approaches. Personally prefer composing in this `{{ }}` construct, but it's up to you.
https://youtu.be/_zBwRDEFMRY 🙂

Reply:
I prefer the first variant, as in the latter one is IMHO easier to miss the hash when reading the line.